content/v2.0/reference/storage-engine.md

```diff
@@ -0,0 +1,157 @@
+It ensures that written data does not disappear in an unexpected failure.
+When a client sends a write request, the following occurs:
+
+1. Write request is appended to the end of the WAL file.
+2. `fsync()` the data to the file.
+3. Update the in-memory database.
+   <!-- 3. Update the in-memory CACHE? -->
+4. Return success to caller.
+
+`fsync()` takes the file and pushes pending writes all the way through any buffers and caches to disk.
+As a system call, `fsync()` has a kernel context switch which is expensive _in terms of time_ but guarantees your data is safe on disk.
+
+{{% note%}}
+To `fsync()` less frequently, batch your points (send in ~2000 points at a time).
+{{% /note %}}
+
+When the storage engine restarts, the WAL file is read back into the in-memory database.
+InfluxDB then snswer requests to the `/read` endpoint.
+
+<!-- ===== V1 material -->
+<!-- TODO is this still true? -->
+<!-- On the file system, the WAL is made up of sequentially numbered files (`_000001.wal`). -->
+<!-- The file numbers are monotonically increasing and referred to as WAL segments. -->
+<!-- When a segment reaches 10MB in size, it is closed and a new one is opened. Each WAL segment stores multiple compressed blocks of writes and deletes. -->
+<!-- Each entry in the WAL follows a [TLV standard](https://en.wikipedia.org/wiki/Type-length-value) with a single byte representing the type of entry (write or delete), a 4 byte `uint32` for the length of the compressed block, and then the compressed block. -->
+
+{{% note%}}
+Once you receive a response to a write request, your data is on disk!
+{{% /note %}}
+
+## Cache
+
+Queries to the storage engine will merge data from the Cache with data from the TSM files.
+Queries execute on a copy of the data that is made from the cache at query processing time.
+This way writes that come in while a query is running won’t affect the result.
+
+The cache is an in-memory copy of data points current stored in the WAL.
+Points are organized by the key, which is the measurement, tag set, and unique field.
+Each field is stored in its own time-ordered range.
+Data is not compressed in the cache.
+The cache is recreated on restart by re-reading the WAL files on disk back into memory.
+The cache is queried at runtime and merged with the data stored in TSM files.
+Deletes sent to the Cache will clear out the given key or the specific time range for the given key.
+
+## Time-Structured Merge Tree (TSM)
+
+To efficiently compact and store data,
+the storage engine groups field values by [series](/v2.0/reference/key-concepts/data-elements/#series) key,
+and then orders those field values by time.
+
+The storage engine uses a **Time-Structured Merge Tree** (TSM) data format.
+TSM files store compressed series data in a columnar format.
+Within a series, we store only , which is more efficient.
+To improve efficiency, the storage engine only stores differences between values in a series.
```

Comment:
...only stores differences (or deltas) between...

I think "deltas" will be more clear to engineers who will likely the be ones reading this type of documentation.